soh/soh/Enhancements/randomizer/randomizerTypes.h

```diff
@@ -1182,6 +1183,14 @@ typedef enum {
     RO_BRIDGE_MAX,
 } RandoOptionRainbowBridge;
 
+//Reward Options settings (Standard rewards, Greg as reward, Greg as wildcard)
+typedef enum {
+    RO_REWARD_STANDARD,
+    RO_REWARD_GREG,
+    RO_REWARD_WILDCARD,
+    RO_REWARD_MAX,
```

Comment:
Mentioning here too that MAX won't be necessary after #2576

Reply:
Until that is merged, I don't really feel comfortable making all these changes, as right now, it would just break the functionality of my PR, so I would have no way to test it to make sure it's working anymore. At least from what I understand of how the current code works. I would need all the changes of your PR first in order for it to work, correct?